Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -144,17 +145,20 @@
                                         x:Name="ItemLocationName"
                                         Margin="0,0,0,8"
                                         VerticalAlignment="Center"
+                                        x:Phase="1"
```

Comment:
Could you give this the same treatment as RecentFileList? Remove "Phase 1" and drecrease "Phase 2"?